### PR TITLE
#4475 added max-height for expanded state of dropdown

### LINKED ISSUE #4475: Dropdowns don't have scroll bar on website
**Description**:
Dropdowns don't have scroll bar on website

**Expected behavior**:
Dropdowns have scroll bars

**To Reproduce**:
1. Open PDP 
2. Add product to cart
3. Open Checkout
4. Open Country dropdown

**Actual result**:
Dropdowns don't have scroll bar
Screenshot 2022-03-16 at 11 36 57

**Notes:**
The same situation PDP, PLP, Address book, Checkout

### PR DESCRIPTION
**Pull request by AzizKHAN030**
**Related issue(s):**
* Fixes #4475

**Problem:**
* Dropdown select didn't have max-height so it didn't scroll

**In this PR:**
* Added max-height 300px

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
1. Now height of dropdown is 300px. Add scroll bar if dropdown have 4+ options (It will be about 200px)
Screenshot 2022-03-21 at 14 27 57

**Comment by AzizKHAN030** [comment]
Access this pull request here: https://pr-4509.d25kd40decjpue.amplifyapp.com/

**Comment by aleksandrakorolova** [comment]
2. The last option is partially showing in currency switcher
Screenshot 2022-03-22 at 13 54 32

**Review by riha112 — APPROVED** [review]

File: packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss [hunk 1]
```diff
@@ -79,7 +79,7 @@ $select-arrow-width: 14px !default;
         &_isExpanded {
             @include desktop {
                 z-index: 15;
-                max-height: fit-content;
+                max-height: $select-option-height * 5;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss:82]: wrap with a calc

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**